CONTRIBUTING.md

```diff
@@ -0,0 +1,43 @@
+### How to contribute:
+
+It's recommended that you first contact ctsm-software@ucar.edu to discuss the changes or additions that you propose.
+Work with them to think about the design and implemention of the changes as well as the planning
+and scheduling for the change to come into CTSM.
+
+It's also recommened that you join the ctsm-dev@ucar.edu google groups in order 
+to be informed about the science that is happening in CTSM as well as the latest develepments and tags.
+The ctsm-dev group is here:
+
+https://groups.google.com/a/ucar.edu/forum/#!forum/ctsm-dev
+
+Use the help from the WIKI below to setup a fork and personal branch in github to put your developments
+on and keep up to date with the main development CTSM. Once the changes are sufficiently advanced you
+can form a Pull Request on github under
+
+https://github.com/ESCOMP/ctsm/pulls
+
+To show your proposed changes and start getting feedback on your code changes. This also allows your
```

Comment:
Move this first sentence fragment "To show..." to the above text (before the link), because it feels awkward here.